src/snowflake/cli/plugins/nativeapp/codegen/snowpark/models.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import List, Optional
+
+from pydantic import Field
+from snowflake.cli.api.project.schemas.snowpark.callable import _CallableBase
+from snowflake.cli.api.project.schemas.updatable_model import IdentifierField
+
+
+class ExtensionFunctionTypeEnum(str, Enum):
+    PROCEDURE = "PROCEDURE"
+    FUNCTION = "FUNCTION"
+    TABLE_FUNCTION = "TABLE_FUNCTION"
+    AGGREGATE_FUNCTION = "AGGREGATE_FUNCTION"
```

Comment:
Any reason why we need underscores here? Is there somewhere we're using them that would parse incorrectly, or is it more of a defense against these perhaps being used in a future context where they'd be treated as two separate literals?

Reply:
I can remove the underscores so that YAML looks pretty.